### DISCUSSION TITLE
Welcome to regional-mom6 Discussions!

### DISCUSSION THREAD

**Discussion by navidcy** [COSIMA/mom6-regional-scripts#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by helenmacdonald** [comment]
Hello 👋 
I am a regional and coastal ocean modeller, just started at ACCESS-NRI in Canberra. Thank you for sharing this code, it is a great way to get started on regional modelling with mom6.

**Reply by navidcy** [reply]
hi @helenmacdonald! welcome!

**Reply by helenmacdonald** [reply]
Thanks 😄
